src/Form/DataTransformer/ModelToIdPropertyTransformer.php

```diff
@@ -110,15 +112,7 @@ public function reverseTransform($value)
             throw new \UnexpectedValueException(sprintf('Value should be array, %s given.', \gettype($value)));
         }
 
-        foreach ($value as $key => $id) {
-            if ('_labels' === $key) {
-                unset($value[$key]);
-
-                continue;
-            }
-
-            $value[$key] = (string) $id;
```

Comment:
addIdentifiersToQuery supports both `int` and `string` so it's not needed to cast it.